networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/peers/Eth2PeerManager.java

```diff
@@ -287,4 +290,18 @@ public BeaconChainMethods getBeaconChainMethods() {
   private boolean peerIsReady(Eth2Peer peer) {
     return peer.hasStatus();
   }
+
+  public SafeFuture<?> sendGoodbyeToPeers() {
+    return SafeFuture.allOf(
+        connectedPeerMap.values().stream()
+            .map(
+                peer ->
+                    peer.disconnectCleanly(DisconnectReason.SHUTTING_DOWN)
+                        .exceptionally(
+                            err -> {
+                              LOG.debug("Error while sending goodbye to peers", err);
+                              return null;
+                            }))
```

Comment:
Can probably just drop this exceptionally clause as `disconnectCleanly` already has one.